tools/generator/CodeGenerator.cs

```diff
@@ -278,8 +278,10 @@ public static void Run (CodeGeneratorOptions options)
 				apiXmlFile = api_xml_adjuster_output ?? Path.Combine (Path.GetDirectoryName (filename), Path.GetFileName (filename) + ".adjusted");
 				new Adjuster ().Process (filename, SymbolTable.AllRegisteredSymbols ().OfType<GenBase> ().ToArray (), apiXmlFile);
 			}
-			if (only_xml_adjuster)
+			if (only_xml_adjuster) {
+				resolver.Dispose ();
```

Comment:
I thought of a different way to ensure disposal w/o re-indenting hundreds of lines...

Fixed.